docs/pages/careers/staff-engineer-pigment-css.md

```diff
@@ -0,0 +1,126 @@
+
+### The worst parts of this job
+
+- **Shifting context.**
+  You will necessarily have to shift context and dive into a different feature before the current one is done.
+  It may even be in an area of the codebase you're unfamiliar with or don't have a ton of understanding about.
+  It's fun, rewarding work, but it can be very challenging.
+- **We move quickly but don't sacrifice quality**.
+  We ship early, often, and quickly. You may not be initially comfortable with the cadence with which we ship high-quality features and improvements to end-users. By doing so, we sacrifice on solving each problem 100% in exchange for fast feedback. Solving 50-70% of the issue with quality should be enough for any given iteration. Our users quickly tell us when we haven't pushed a solution far enough.
+- **MUI has a large codebase**. You may bang your head against the wall at times, and then write tests to assist future you 😌.
+  The work you will be doing is somewhat unique and idiosyncratic. You probably have not had a similar role before.
+- **In open-source, you're faced with a nonstop stream of bug reports and support requests**. That means you need to develop an intuition for when to ignore something, and when to dig in further.
+
+## Who we're looking for
+
+We're looking for someone with strong core JavaScript skills. More important than specific technical skills though is that you're a strong problem solver who loves to learn. Details matter to you.
+
+### Required
+
+- **Expertise in the modern JavaScript ecosystem**.
+  MUI's open-source projects are built on the shoulders of giants, making use of technologies such as ES2021, TypeScript, Node.js, React, Next.js, Webpack, and Babel.
+- **A track record of demonstrating an eye for product and solving real-world user problems**. If you have a knack for solving problems at the root cause, shipping beautiful user interfaces and intuitive APIs, we want you on our team.
+- **Experience building and shipping production code in a team setting** with a passion for writing tested, performant, and high-quality code.
+- **Strong written and verbal communication skills**.
+  As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
+- **Ability to dive into complex problems**.
+  You should be able to quickly assess, understand, and iterate upon aspects of our codebase.
+- **Ready and willing to ask and answer questions**.
+  If you're comfortable saying you're unsure, asking for help; but equally reaching out to assist others, you'll be an incredible addition to our team. We thrive because of continuous learning. First-time mistakes should be celebrated, not blamed.
+- **Avoid monolithic deliverables**.
+  You scope and stage your work into well-defined milestones to ship.
```

Comment:
```suggestion

```

I'd cut this one — it seems like too much of a practical guideline. If we'd like to keep it, I'd probably change to something along the lines of "You have a great sense of project management" because that's what it essentially is about: iterating fast, incrementally, with clear milestones

Reply:
Seeing this in the job offer makes me feel a bit like it's helpful to differentiate Junior/Mid level to Senior. But if the role is Staff, we need to emphasize on what sets Staff appart as we might see a lot of Senior apply.

Reply:
I have tried this:

> - **Leadership skills**. You will help build and collaborate with the community. You will guide and mentor contributors, steering the community in the right direction.